src/unstract/adapters/llm/palm/src/static/json_schema.json

```diff
@@ -28,7 +28,7 @@
       "type": "number",
       "minimum": 0,
       "multipleOf": 1,
-      "title": "Number of Output Tokens",
+      "title": "Max output tokens",
```

Comment:
Not setting any default here because, when we do not pass anything, llama-index automatically detects the max supported for the model and sets it accordingly.

Reply:
Is this applicable only for Palm?

Reply:
@gaya3-zipstack how will the user know this default value? What will happen if the user exceeds the max default?

Reply:
> @gaya3-zipstack how will the user know this default value? What will happen if the user exceeds the max default?

Ideally llama-index may throw error. However we have not extensively tested this. There are a few cases, where it silently passed and went through @chandrasekharan-zipstack